src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -38,19 +39,21 @@ export function staticKeyToShadowsocksSessionConfig(staticKey: string): Shadowso
 function parseShadowsocksSessionConfigJson(maybeJsonText: string): ShadowsocksSessionConfig | null {
   let sessionConfig;
   try {
-    const {method, password, server: host, server_port: port} = JSON.parse(maybeJsonText);
+    const {method, password, server: host, server_port: port, extra} = JSON.parse(maybeJsonText);
```

Comment:
Let's have an explicit prefix field instead. The dynamic config should align with the SessionConfig.